src/modules/search/utils/filterMap.ts

```diff
@@ -1,28 +1,35 @@
-import { FilterMap, FilterType, QueryParams } from '../../../types/common';
+import { PartialConfig, FilterType, QueryParams } from '../../../types/common';
+import { formatRange } from './formatRange';
+import { groupNumbers, listGroupedNumbers } from '../../../utils/groupNumbers';
 
 const fiveOrMoreItem = '5+ h';
```

Comment:
To follow DDD philosophy this could be renamed to `fiveOrMoreRooms` for clarity

Reply:
Donets